### PR TITLE
[CI TC Fail Fix] test/grpc: correct parallel unit test calls @open sesame 6/10 13:24

### PR DESCRIPTION
**Pull request by myungjoo**
1. Do not fork ssat-api functions
2. Do not sleep constant time. Check the launch process.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: tests/nnstreamer_grpc/runTest.sh [hunk 1]
```diff
@@ -67,6 +67,25 @@ export LD_LIBRARY_PATH=${PATH_TO_PLUGIN_EXTRA}:${LD_LIBRARY_PATH}
 
 NUM_BUFFERS=10
 
+TIMEOUT_SEC=10
+## Check if a pipeline is launched and playing
+function waitformarker {
```
Comment (jaeyun-jung) [inline tests/nnstreamer_grpc/runTest.sh:72]: Q: is it possible to add util script and use common function waitformarker (query and grpc)?

Comment (myungjoo) [reply tests/nnstreamer_grpc/runTest.sh:72]: It'd be better located at SSAT (ssat-api). Because it's in a different repo, let's do it later

**Review by jaeyun-jung — APPROVED** [review]

**Comment by myungjoo** [comment]
@jaeyun-jung PTAL: https://github.com/myungjoo/SSAT/pull/22

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**